content/en/blog/news/five-new-things/index.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2024-11-08
+title: Five New Things in SpinKube
+linkTitle: Five New Things in SpinKube
+description: >
+  Catching up on what's new in SpinKube
+author: The SpinKube Team ([@SpinKube](https://mastodon.social/@SpinKube))
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+Since we publicly [released](/blog/2024/03/13/introducing-spinkube/) SpinKube in March we've been hard at work steadily making it better. Spin Operator [`v0.4.0`](https://github.com/spinkube/spin-operator/releases/tag/v0.4.0), ContainerD shim for Spin [`v0.17.0`](TODO), and `spin kube` plugin [`v0.3.0`](https://github.com/spinkube/spin-plugin-kube/releases/tag/v0.3.0) have all just been released. To celebrate that, here's five new things in SpinKube you should know about.
+
+## Selective Deployments
+
+SpinKube now supports selectively deploying a subset of a Spin apps components. Here's an example of what the YAML for a selectively deployed app might look like:
+
+```yaml
+apiVersion: core.spinkube.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: salutations
+spec:
+  image: "ghcr.io/spinkube/spin-operator/salutations:20241105-223428-g4da3171"
+  executor: containerd-shim-spin
+  replicas: 1
+  components:
+    - hello
+```
+
+We're really excited about this feature because it makes developing microservices easier. Locally keep your whole app together and develop it as a monolith. When you go to production selectively split your app based on the characteristics of each component. For example you might run one component on nodes with GPUs and another on nodes that are closer to the end user.
```

Comment:
```suggestion
We're really excited about this feature because it makes developing microservices easier. Locally develop your application as a monolith in one code base. Then, when you go to production, you can split your app based on the characteristics of each component. For example you run your front end closer to the end user while keeping your backend colocated with your database.
```
I don't like the GPU example -- even though i know i started it -- because we don't support gpu access from spin apps